### PR TITLE
Initial bootnodes

### PR DESCRIPTION
**Pull request by protolambda**
Initial bootnodes, many more to come.

### CODE REVIEW

File: bootnodes.txt [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Eth2 mainnet bootnodes
+# ---------------------------------------
+# 1. Tag nodes with maintainer
+# 2. Keep nodes updated
+# 3. Review PRs: check ENR duplicates, fork-digest, connection.
+
+# Teku team's bootnode
+enr:-KG4QOtcP9X1FbIMOe17QNMKqDxCpm14jcX5tiOE4_TyMrFqbmhPZHK_ZPG2Gxb1GE2xdtodOfx9-cgvNtxnRyHEmC0ghGV0aDKQ9aX9QgAAAAD__________4JpZIJ2NIJpcIQDE8KdiXNlY3AyNTZrMaEDhpehBDbZjM_L9ek699Y7vhUJ-eAdMyQW_Fil522Y0fODdGNwgiMog3VkcIIjKA
```
Comment (ajsutton) [inline bootnodes.txt:8]: Confirm this is correct for Teku.
We've also added a Sydney based bootnode: 
```
enr:-KG4QDyytgmE4f7AnvW-ZaUOIi9i79qX4JwjRAiXBZCU65wOfBu-3Nb5I7b_Rmg3KCOcZM_C3y5pg7EBU5XGrcLTduQEhGV0aDKQ9aX9QgAAAAD__________4JpZIJ2NIJpcIQ2_DUbiXNlY3AyNTZrMaEDKnz_-ps3UUOfHWVYaskI5kWYO_vtYMGYCQRAR3gHDouDdGNwgiMog3VkcIIjKA
```

File: bootnodes.txt [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+# Prylab team's bootnodes
+enr:-Ku4QImhMc1z8yCiNJ1TyUxdcfNucje3BGwEHzodEZUan8PherEo4sF7pPHPSIB1NNuSg5fZy7qFsjmUKs2ea1Whi0EBh2F0dG5ldHOIAAAAAAAAAACEZXRoMpD1pf1CAAAAAP__________gmlkgnY0gmlwhBLf22SJc2VjcDI1NmsxoQOVphkDqal4QzPMksc5wnpuC3gvSC8AfbFOnZY_On34wIN1ZHCCIyg
+enr:-Ku4QP2xDnEtUXIjzJ_DhlCRN9SN99RYQPJL92TMlSv7U5C1YnYLjwOQHgZIUXw6c-BvRg2Yc2QsZxxoS_pPRVe0yK8Bh2F0dG5ldHOIAAAAAAAAAACEZXRoMpD1pf1CAAAAAP__________gmlkgnY0gmlwhBLf22SJc2VjcDI1NmsxoQMeFF5GrS7UZpAH2Ly84aLK-TyvH-dRo0JM1i8yygH50YN1ZHCCJxA
+enr:-Ku4QPp9z1W4tAO8Ber_NQierYaOStqhDqQdOPY3bB3jDgkjcbk6YrEnVYIiCBbTxuar3CzS528d2iE7TdJsrL-dEKoBh2F0dG5ldHOIAAAAAAAAAACEZXRoMpD1pf1CAAAAAP__________gmlkgnY0gmlwhBLf22SJc2VjcDI1NmsxoQMw5fqqkw2hHC4F5HZZDPsNmPdB1Gi8JPQK7pRc9XHh-oN1ZHCCKvg
```
Comment (ajsutton) [inline bootnodes.txt:14]: These match what we've added to Teku.  They do have a 0 tcp port in the multiaddr but IP is sane and has a UDP port of 9000.

**Review by ajsutton — DISMISSED** [review]
LGTM.

**Comment by AgeManning** [comment]
Lighthouse's bootnodes reporting in:
```
enr:-IS4QLkKqDMy_ExrpOEWa59NiClemOnor-krjp4qoeZwIw2QduPC-q7Kz4u1IOWf3DDbdxqQIgC4fejavBOuUPy-HE4BgmlkgnY0gmlwhCLzAHqJc2VjcDI1NmsxoQLQSJfEAHZApkm5edTCZ_4qps_1k_ub2CxHFxi-gr2JMIN1ZHCCIyg
enr:-IS4QDAyibHCzYZmIYZCjXwU9BqpotWmv2BsFlIq1V31BwDDMJPFEbox1ijT5c2Ou3kvieOKejxuaCqIcjxBjJ_3j_cBgmlkgnY0gmlwhAMaHiCJc2VjcDI1NmsxoQJIdpj_foZ02MXz4It8xKD7yUHTBx7lVFn3oeRP21KRV4N1ZHCCIyg
```

**Comment by protolambda** [comment]
@AgeManning it looks like your ENRs do not have an `eth2` field (no fork digest!). They may get rejected by eth2 nodes.

File: bootnodes.txt [hunk 3]
```diff
@@ -0,0 +1,26 @@
+# Eth2 mainnet bootnodes
+# ---------------------------------------
+# 1. Tag nodes with maintainer
+# 2. Keep nodes updated
+# 3. Review PRs: check ENR duplicates, fork-digest, connection.
+
+# Teku team's bootnode
```
Comment (ajsutton) [inline bootnodes.txt]: If its useful to note, this one is in Ohio.

**Review by ajsutton — DISMISSED** [review]
Confirmed these bootnodes work with Teku. LGTM.

**Comment by AgeManning** [comment]
Corrected Lighthouse bootnodes:
```
enr:-Jq4QN6_FzIYyfJET9hiLcGUsg_EVOwCQ4bwsBwe0S4ElrfXUXufSYLtQAHU9_LuO9uice7EAaLbDlMK8QEhtyg8Oh4BhGV0aDKQtTA_KgAAAAD__________4JpZIJ2NIJpcIQDGh4giXNlY3AyNTZrMaECSHaY_36GdNjF8-CLfMSg-8lB0wce5VRZ96HkT9tSkVeDdWRwgiMo
enr:-Jq4QMOjjkLYSN7GVAf_zBSS5c_MokSPMZZvmjLUYiuHrPLHInjeBtF1IfskuYlmhglGan2ECmPk89SRXr4FY1jVp5YBhGV0aDKQtTA_KgAAAAD__________4JpZIJ2NIJpcIQi8wB6iXNlY3AyNTZrMaEC0EiXxAB2QKZJuXnUwmf-KqbP9ZP7m9gsRxcYvoK9iTCDdWRwgiMo
```

Fork Digest:  `b5303f2a`

**Comment by kdeme** [comment]
Nimbus bootstrap nodes:

```
enr:-LK4QLU5_AeUzZEtpK8grqPo4EmX4el3ochu8vNNoXX1PrBjYfn8ksjeQ1eFtbL7ywMau9k_7BBQGmO26DHWgngkBCgBh2F0dG5ldHOI__________-EZXRoMpC1MD8qAAAAAP__________gmlkgnY0gmlwhAN7_O-Jc2VjcDI1NmsxoQKH1zg2Fge8Q6Zf-rLFbjGEtgvVbmDXqFVLxqquJcguFIN0Y3CCI4yDdWRwgiOM
enr:-LK4QLjSKc09WkFZ5Pa1UF3KPkt3ieTZ6B7F6iDL_chyniP5NVDl10aGIu-pL9mbwZ47GM3RN63eGHPsw-MTLSYcz74Bh2F0dG5ldHOI__________-EZXRoMpC1MD8qAAAAAP__________gmlkgnY0gmlwhDQ7fI6Jc2VjcDI1NmsxoQJDU6zzDlUDgUqFSzoIuP9bWu097k2d7X4eHoJTGhbphoN0Y3CCI4yDdWRwgiOM
```

**Comment by kdeme** [comment]
I noticed that several of the bootnodes have no `tcp` field set, but however do have the `eth2` field set. 
I'm guessing that these are "discovery only" nodes, without eth2 functionality, but then I'm curious why the `eth2` field is still set (and is that even according to spec?). 

I can only assume that some(?) clients filter out nodes without that field directly from there kademlia routing table? I don't really think this is/was the design goal of having ENRs, that is, to separate networks. 



> @AgeManning it looks like your ENRs do not have an `eth2` field (no fork digest!). They may get rejected by eth2 nodes.

This seems to confirm this?

**Comment by AgeManning** [comment]
Yes. The bootnodes we have provided are "discovery only" nodes. 

Its a good point about the `eth2` field. Lighthouse searches for peers with the specific `eth2` field, however during the searches we should still add non-eth2 ENRs to our routing table. We attempt to connect to other eth2 nodes that have a matching `eth2` field and valid tcp port. 

I guess for discovery-only nodes the eth2 field is not entirely important for us, but maybe for others.

**Review by ajsutton — APPROVED** [review]

**Review by terencechain — APPROVED** [review]

**Pull request merged**